modules/claims_api/app/controllers/claims_api/v2/veterans/claims_controller.rb

```diff
@@ -20,10 +19,25 @@ def index
           render json: claims
         end
 
+        def show
+          bgs_claim = bgs_service.benefit_claims.find_claim_details_by_claim_id(claim_id: params[:id]) || {}
```

Comment:
we need to be able to accept our guid's for claims still in pending state as well